src/github.com/matrix-org/dendrite/common/log.go

```diff
@@ -70,14 +70,27 @@ func SetupStdLogging() {
 // SetupHookLogging configures the logging hooks defined in the configuration.
 // If something fails here it means that the logging was improperly configured,
 // so we just exit with the error
-func SetupHookLogging(hooks []config.Hook, componentName string) {
+func SetupHookLogging(hooks []config.LogrusHook, componentName string) {
 	for _, hook := range hooks {
+
+		// Ensures we received a proper logging level
```

Comment:
unless you're going to write a long comment block ("The following code ensures we received a proper logging level by doing the X and then the Y"), comments should be written in the imperative: s/Ensures/Check/ please.